flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CodeAnalyzer.java

```diff
@@ -367,11 +370,11 @@ private void checkForNewConnection(NewExpressionNode newExpressionNode,
         startNode(FlowNode.Kind.NEW_CONNECTION)
                 .metadata().description(nodeTemplate.metadata().description()).stepOut()
                 .codedata()
-                    .org(nodeTemplate.codedata().org())
-                    .module(nodeTemplate.codedata().module())
-                    .object(nodeTemplate.codedata().object())
-                    .symbol(nodeTemplate.codedata().symbol())
-                    .stepOut()
```

Comment:
These are intended formatting done by the IntelliJ for faceted builders. Let's preserve them

Reply:
updated